Sources/Paywalls/PaywallCacheWarming.swift

```diff
@@ -90,6 +90,15 @@ actor PaywallCacheWarming: PaywallCacheWarmingType {
 
         Logger.verbose(Strings.paywalls.warming_up_images(imageURLs: imageURLs))
 
+        // Preferred method - load with FileRepository
+        let fileRepository = FileRepository()
+        for url in imageURLs {
+            Task {
+                _ = try await fileRepository.generateOrGetCachedFileURL(for: url)
+            }
```

Comment:
Actually, let's put this in the other for loop so you don't need to loop over all the images twice, and since the url session cache that is in the image fetcher is the backup, you can do a `try? await` IMO